storage-xray-udp/src/main/java/zipkin2/storage/xray_udp/UDPMessageEncoder.java

```diff
@@ -55,8 +57,13 @@
         || span.kind() != Span.Kind.SERVER && span.kind() != Span.Kind.CONSUMER) {
       writer.name("type").value("subsegment");
       if (span.kind() != null) writer.name("namespace").value("remote");
+      writer.name("name").value(span.remoteServiceName() == null ? "" : span.remoteServiceName());
```

Comment:
do we need to write empty name? or is leaving out name tolerable? If possible, I'd prefer to not write placeholder values

Reply:
I will test it and let you know.

Reply:
when it is null or empty string, it ignores span so it does not appear at all. There is two possibility when it is null or empty:

1. name it "unknown"
2. ignore span

Which one seems better for you @adriancole ?